controllers/deleteComment-controller.js

```diff
@@ -0,0 +1,13 @@
+const { removeComment } = require("../models/removeComment-model");
+
+exports.deleteComment = (req, res, next) => {
+  const { params } = req;
+  const id = params.comment_id;
+  removeComment(id)
+    .then((rows) => {
+      res.status(204).send({});
```

Comment:
We don't want to be sending anything back here, so either no argument in the .send() would work, or you can make use of res.sendStatus().